styleguide/source/assets/scss/02-molecules/_hero-evergreen.scss

```diff
@@ -0,0 +1,88 @@
+.hero-evergreen {
+  display: flex;
+  flex-flow: column wrap;
+
+  @include breakpoint($bp-med) {
+    flex-flow: row nowrap;
+  }
+
+  &__mobile-image {
+    display: block;
+
+    @include breakpoint($bp-small) {
+      display: none;
+    }
+  }
+
+  &__aside {
+    @include gutter-all(padding, $gutter / 2, $gutter-mobile / 2);
+    background: $purple;
+    flex-basis: 33.33%;
+    color: $white;
+
+    &--full-width {
+      flex-basis: 100%;
+    }
+  }
+
+  &__background {
+    flex-basis: 66.66%;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    background-size: cover;
+
+    @include breakpoint($bp-small max-width) {
```

Comment:
@include breakpoint($bp-large min-width) {
  background: none;
}

let's not use importants

If you use the mobile first approach then the last breakpoint would be large

Reply:
Unfortunately, this doesn't do what I need it to do. The background image is set inline in the twig template, but I need `background: none` only on small size. The only way that I know of to override the inline CSS is to add `!important`. I used the `breakpoint($bp-small max-width)` because going with a mobile first approach, I would not be able to then override the `!important` needed for the small size (the background image is set in the template and I cannot pull that into the CSS). Do you know of a better way to accomplish this?

Reply:
I see. I guess this way will work.

Reply:
Great solution! It is now implemented.